src/sentry/features/__init__.py

```diff
@@ -56,6 +56,7 @@
 default_manager.add("organizations:alert-filters", OrganizationFeature)
 default_manager.add("organizations:api-keys", OrganizationFeature)
 default_manager.add("organizations:app-store-connect", OrganizationFeature, True)
+default_manager.add("organizations:app-store-connect-multiple", OrganizationFeature, True)
```

Comment:
I'm not 100% familiar with this, but if this is not controlled by flagr, does this have to be an entity feature?